Userland/Libraries/LibJS/Tests/builtins/Array/Array.prototype.slice.js

```diff
@@ -38,10 +38,7 @@ test("basic functionality", () => {
     expect(slice).toEqual(["hello", "friends", "serenity", 1]);
 });
 
-// FIXME: These tests are currently skipped because an invalid array length in this case is 2**32 or above.
-//        The codebase currently uses size_t for lengths, which is currently the same as u32 when building for Serenity.
-//        This means these lengths wrap around to 0, making the test not work correctly.
-test.skip("Invalid lengths", () => {
+test("Invalid lengths", () => {
     var length = Math.pow(2, 32);
```

Comment:
This is really unrelated but I forgot to do this in previous PRs

Reply:
We're still using size_ts and this still fails, do it should still be skipped..

Reply:
I keep forgetting it also runs in Serenity. You're right

Reply:
Yeah, we gotta shave the LibJS size_t yak at some point